volatility3/framework/symbols/linux/utilities/modules.py

```diff
@@ -10,44 +22,51 @@
     exceptions,
     objects,
 )
+
 from volatility3.framework.objects import utility
 from volatility3.framework.symbols.linux import extensions
 
 vollog = logging.getLogger(__name__)
 
 
-class Modules(interfaces.configuration.VersionableInterface):
-    """Kernel modules related utilities."""
+class ModuleInfo(NamedTuple):
```

Comment:
I think this lived inside of the `Modules` class, so that it was part of the versioning of the Modules code.  If it's defined in the outside it's not tied together and has to work for all time as is...

Reply:
Yes I am aware. If that every changes or breaks I will have to start from scratch anyway. It is used between both Modules and ModuleGatherers, so these versions would bump too.

Reply:
I want plugins that do the rootkit checks to version both. Ones that don't can just do Modules and not get version bumped or changed when rootkit related stuff changes. The "everything in on modules" makes versioning very painful.

Reply:
Ok, but... the gatherers should be versioned under something less than the framework (at the moment they're just floating free in this module.  They could each be individually versioned, but then the "scan everything list" would need bumping every time one of them got bumped?  Happy to go that route if you think it's the right one?

Reply:
Ah yea, that level of granularity is nice. I will work that in.